### ISSUE TITLE
TWIW newsletter links should use enough contrast

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.io#2]
At the moment, the links in the This week in Wagtail newsletter don't provide enough contrast to pass accessibility checks.

Screenshot:

<img width="577" alt="twiw-link-colors" src="https://user-images.githubusercontent.com/877585/55971463-6f52b400-5c79-11e9-8b9c-275efe6c0d9c.png">

The font size should be increased, or the colors should be changed to ones that are higher-contrast over white.

Relevant template (thanks @kaedroho for the find!): https://github.com/wagtail/wagtail.io/blob/master/wagtailio/newsletter/templates/newsletter/newsletter_page.html

---

Initially raised [on Slack](https://wagtailcms.slack.com/archives/C4G0STFQX/p1554374519002800).
